src/components/ui-shell/_side-nav.scss

```diff
@@ -56,7 +56,8 @@
   // Include the fixed variant here so components can render in both expandable
   // and fixed side navs
   .#{$prefix}--side-nav--fixed &,
-  .#{$prefix}--side-nav--expanded & {
+  .#{$prefix}--side-nav--expanded &
+  .#{$prefix}--side-nav--focused & {
```

Comment:
`:focus`?

Reply:
I think it would lose the style once a nav item is focused. `:focus-within` would be better but doesn’t support edge/ie

Reply:
Thanks @vpicone! Your comment made me understand that the original issue is a request for "expand on focus", which requires bigger discussion among the team to see the desired behavior. Wondering if there has been a consideration for keyboard interaction with the expando.

Reply:
Right now there is a bug when you tab with keyboard through the sideNav. You can even focus items that are not visible which is what prompted my initial attempts to correct it.